packages/eslint-plugin-next/lib/index.js

```diff
@@ -42,6 +43,7 @@ module.exports = {
         '@next/next/no-unwanted-polyfillio': 'warn',
         // errors
         '@next/next/inline-script-id': 'error',
+        '@next/next/next-head-key': 'error',
```

Comment:
```suggestion
        '@next/next/next-head-key': 'off',
```

`next/head` is smart enough to deduplicate tags in most situations. This rule should not be enabled by default, see https://github.com/vercel/next.js/issues/39491#issuecomment-1212148074